app/views/coronavirus_pages/reorder.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus topic pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "Coronavirus (COVID-19)"
+    }
+  ]
+
+%>
+
+<% content_for :title, "Coronavirus (COVID-19)" %>
+<% content_for :context, "Reorder #{@coronavirus_page.sections_title.downcase} accordion" %>
+
+<div class="covid-manage-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_markdown "Use the up/down buttons on the right to reorder the sections, or click and hold on a section to reorder using drag and drop." %>
+    <%= render "coronavirus_pages/sub_sections_summary_list", { reorder: true } %>
```

Comment:
If there was separate partial for the reorder view, this could become something like:
```irb
<%= render "coronavirus_pages/sub_sections_reorder_list" %>
```